frontend/common/src/components/Combobox/Combobox.tsx

```diff
@@ -0,0 +1,249 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import { FieldError, RegisterOptions, useFormContext } from "react-hook-form";
+import { Combobox as ComboboxPrimitive } from "@headlessui/react";
+import debounce from "lodash/debounce";
+
+import InputUnsaved from "../inputPartials/InputUnsaved/InputUnsaved";
+import InputError from "../inputPartials/InputError/InputError";
+import { useFieldState, useFieldStateStyles } from "../../helpers/formUtils";
+import { Scalars } from "../../api/generated";
+
+import Actions from "./Actions";
+import Label from "./Label";
+import NoOptions from "./NoOptions";
+
+import "./combobox.css";
+
+export interface Option {
+  value: string;
+  label: React.ReactNode;
+}
+
+export interface ComboboxProps
```

Comment:
Could you document all these props?  Most of them are obvious but I think it's important for components that will be widely reused.

Reply:
Oops, I totally forgot that. Thanks for reminding me!

Added with [document props for combobox](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5064/commits/543000f2a9a6bc339c852a064e10646717e728e2)